config/initializers/01_obis_setup.rb

```diff
@@ -90,6 +90,11 @@
   USE_FEEDBACK_LINK                         = application_config['use_feedback_link'] || false
   FEEDBACK_LINK                             = application_config['feedback_link'] || "#"
   NAVBAR_LINKS                              = application_config['navbar_links'] || {}
+  USE_CAS                                   = application_config['use_cas'] || false
```

Comment:
@loganfreeman Can you add the corresponding constants to the application.yml.example file:
`use_cas`
`use_cas_only`
Also note: We will need to add these to the OS Settings branch dataset

Reply:
Kyle:

I added use_cas and use_cas_only to application.yml.example

Reply:
Thanks!